README.md

```diff
@@ -235,7 +235,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
     * [coala](http://coala-analyzer.org/) - Language independent and easily extendable code analysis application.
     * [code2flow](https://github.com/scottrogowski/code2flow) - Turn your Python and JavaScript code into DOT flowcharts.
     * [pycallgraph](https://github.com/gak/pycallgraph) - A library that visualises the flow (call graph) of your Python application.
-    * [pysonar2](https://github.com/yinwang0/pysonar2) - A type inferencer and indexer for Python.
+    * [pysonar2](https://github.com/DreamLinuxer/pysonar2) - A type inferencer and indexer for Python.
```

Comment:
As far as I know, the author decided to unpublish all his open source libraries, maybe we should remove it.
http://www.yinwang.org/blog-cn/2017/04/18/close-source